sdk/tests/deqp/functional/gles3/es3fNegativeVertexArrayApiTests.js

```diff
@@ -544,10 +545,12 @@ goog.scope(function() {
             this.expectError(gl.INVALID_VALUE);
```

Comment:
After the revision, all these cases above in draw_arrays_instanced_invalid_program has been covered in draw_arrays_instanced. So I would like to remove these cases directly. 

So does many other cases in draw_arrays_instanced_invalid_program and draw_elements_instanced_invalid_program.